HelpSource/Guides/OSX_Standalones.schelp

```diff
@@ -0,0 +1,286 @@
+
+This set of scripts creates a SuperCollider-based
+Standalone app in a few basic steps:
+
+1. copy the open SuperCollider.app or SC3-derived standalone.
+
+2. fix the copy's info files so it uses its own userAppSupportDir
+
+3. test that Platform.userAppSupportDir is independent in the new app!
+
+Then the adaptation to the specific project begins:
+
+4. transfer active quarks and extensions to the new app
+
+5. copy the project folder used to the new app
+
+6. make an internal startupFile for it
+
+subsection:: What to prepare
+
+* have your system ready with all desired quarks installed,
+and a working startup file. (can be a very simple test setup)
+
+* read each section, then try evaluating it at once;
+ in case of problems, evaluate line by line,
+ and tell me where which problems happen
+
+* when you get to the wakeup kiss, share and enjoy!
+
+subsection:: Naming, copying, and fixing up the new app
+
+code::
+// 0 - define name and location of new app + helper functions
+(
+// define the name for the new app and its location:
+~newAppName = "Utopia_GenComp_39d";
+~newAppLocation = "~/Desktop".standardizePath;
+
+// some helper functions
+~readText = { |path|
+	var f = File(path, "r"); var t = f.readAllString; f.close; t;
+};
+~writeText = { |path, string| File(path, "w").write(string).close; };
+~copyFolder = { |orig, dest|
+	var str = "cp -r" + quote(orig) + quote(dest);
+	str.postcs;
+	unixCmd(str);
+};
+
+// make some needed paths and folders
+~pathToThisApp = String.scDir.dirname.dirname;
+~thisAppName = ~pathToThisApp.basename.splitext.first;
+~pathToNewApp = ~newAppLocation +/+ ~newAppName ++ ".app";
+~newAppResDir = ~pathToNewApp +/+ "Contents/Resources";
+~newAppSupportDir = Platform.userConfigDir.replace(~thisAppName, ~newAppName);
+);
+
+
+(
+// 1. make a copy of thisApp with the newAppName:
+// can take a while on slow harddisks.
+// wait for RESULT = 0 to show!
+
+if (File.exists(~pathToNewApp).not) {
+	~copyFolder.(~pathToThisApp, ~pathToNewApp);
+	File.mkdir(~newAppSupportDir);
+} { warn("% already exists.".format(~pathToNewApp)) };
+);
+
+(
+// 2. fixups in the new app:
+// a. fix the Info.plist file by replacing the name:
+~infoPlistPath = ~pathToNewApp +/+ "Contents/Info.plist";
+// get its string, replace the SC names, write again
+~infoString = ~readText.(~infoPlistPath);
+
+"replacing app name in plist file at these locations: ".post;
+~infoString.findAll(~thisAppName).postln;
+
+~newInfoString = ~infoString.replace(~thisAppName,  ~newAppName );
+~writeText.(~infoPlistPath, ~newInfoString);
+// ~readText.(~infoPlistPath);
+
+// b. rename the binary file:
+"renaming macos binary to: ".post;
+// fixups in the new app - 2. rename the binary inside the app folder
+unixCmd("mv" + quote(~pathToNewApp +/+ "Contents/MacOS/" +/+ ~thisAppName)
+	+ quote(~pathToNewApp +/+ "Contents/MacOS/" ++ ~newAppName).postln);
+);
+
+
+// 3. *** time for the first wakeup kiss *** //
+unixCmd("open" + ~pathToNewApp);
+
```

Comment:
I got lots of 
```
ERROR: duplicate Class found: 'BinaryOpXStream' 
[...]
ERROR: There is a discrepancy.
numClassDeps 3482   gNumClasses 4752
Library has not been compiled successfully.
Library has not been compiled successfully.
```

maybe because I have a custom `LanguageConfig` file?

so I cannot test further...